packages/itwin/property-grid/src/types.ts

```diff
@@ -30,13 +37,19 @@ export interface PropertyGridProps {
   isOrientationFixed?: boolean;
   enableFavoriteProperties?: boolean;
   favoritePropertiesScope?: FavoritePropertiesScope;
+  customOnDataChanged?: (
+    dataProvider: IPresentationPropertyDataProvider
+  ) => Promise<void>;
+  actionButtonRenderer?: ActionButtonRenderer;
```

Comment:
Shouldn't this have been an array?

Reply:
I'm only passing in the one renderer, but I can make it an array just in case for future uses (plus I know the virtualized property grid takes an array).